packages/speech/src/index.js

```diff
@@ -0,0 +1,816 @@
+ *       },
+ *       {
+ *         transcript: 'Result 2b',
+ *         confidence: 0.80
+ *       },
+ *       ...
+ *     ]
+ *   }
+ * ];
+ *
+ * //-
+ * // Default output.
+ * //-
+ * Speech.formatResults_(resultSets);
+ * // 'Result 1a Result 2a'
+ *
+ * //-
+ * // Verbose output.
+ * //-
+ * Speech.formatResults_(resultSets, true);
+ * // [
+ * //   {
+ * //     transcript: 'Result 1a',
+ * //     confidence: 70,
+ * //     alternatives: [
+ * //       {
+ * //         transcript: 'Result 1b',
+ * //         confidence: 60
+ * //       },
+ * //       ...
+ * //     ]
+ * //   },
+ * //   {
+ * //     transcript: 'Result 2a',
+ * //     confidence: 90,
+ * //     alternatives: [
+ * //       {
+ * //         transcript: 'Result 2b',
+ * //         confidence: 80
+ * //       },
+ * //       ...
+ * //     ]
+ * //   }
+ * // ]
+ */
+Speech.formatResults_ = function(resultSets, verboseMode) {
+  function multiplyScores(result) {
+    if (is.defined(result.confidence)) {
+      result.confidence *= 100;
+    }
+
+    if (is.defined(result.stability)) {
+      result.stability *= 100;
+    }
+
+    return result;
+  }
+
+  var verboseResultSets = resultSets
+    .map(function(resultSet) {
+      resultSet = extend(true, {}, resultSet);
+
+      var mostProbableResult = multiplyScores(resultSet.alternatives.shift());
+
+      resultSet.transcript = mostProbableResult.transcript;
+
+      if (is.defined(mostProbableResult.confidence)) {
+        resultSet.confidence = mostProbableResult.confidence;
+      }
+
+      if (is.defined(mostProbableResult.stability)) {
+        resultSet.stability = mostProbableResult.stability;
+      }
+
+      resultSet.alternatives = resultSet.alternatives.map(multiplyScores);
+
+      return resultSet;
+    });
+
+  if (!verboseMode) {
+    return verboseResultSets.map(prop('transcript')).join(' ');
+  }
+
+  return verboseResultSets;
+};
+
+/**
+ * Perform bidirectional streaming speech-recognition: receive results while
+ * sending audio.
+ *
+ * @resource [`StreamingRecognize` API Reference]{@link https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.Speech.StreamingRecognize}
+ * @resource [`StreamingRecognizeRequest` API Reference]{@link https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognizeRequest}
+ *
+ * @param {object} config - A `StreamingRecognitionConfig` object. See
+ *     [`StreamingRecognitionConfig`](https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognitionConfig).
+ * @param {boolean=} config.verbose - Enable verbose mode for a more detailed
+ *     response. See the examples below. Default: `false`.
+ *
+ * @example
+ * var fs = require('fs');
+ *
+ * //-
+ * // See <a href="https://cloud.google.com/speech/reference/rpc/google.cloud.speech.v1beta1#google.cloud.speech.v1beta1.StreamingRecognitionConfig">
```

Comment:
Can we add this as a `@resource` instead? The current location kind of makes it seem out of place. 
<img width="809" alt="screen shot 2016-09-12 at 12 51 28 pm" src="https://cloud.githubusercontent.com/assets/1707267/18444558/ad9bb590-78e7-11e6-8cbe-2e7fbde8e43c.png">